### PR TITLE
Reloader: Use ScopeMeasure and apply some small refactoring/cleanup

### PR DESCRIPTION
**Pull request by dennisschagt**
I noticed that the `Reloader` class was still using manual calculations with `time()` instead of `ScopeMeasure`.
While fixing that, I noticed some related code which could use some improvement.

### CODE REVIEW

File: include/reloader.h [hunk 1]
```diff
@@ -89,6 +82,15 @@ class Reloader {
 	/// notification will contain \a msg passed.
 	void notify(const std::string& msg);
 
+protected:
```
Comment (Minoru) [inline include/reloader.h]: Class members are private by default, so this commit effectively relaxed the access. No classes inherit from `Reloader`, so the point is kind of moot, but still: why change the access level?

Comment (dennisschagt) [reply include/reloader.h]: My bad, will fix it and force-push this small change.

**Comment by dennisschagt** [comment]
In case it is useful, the output of `git range-diff` for the force-pushed changes:
image

**Comment by Minoru** [comment]
Thank you!

**Pull request merged**